projects/metis/src/app/_services/authentication.service.ts

```diff
@@ -4,9 +4,11 @@ import { Router } from '@angular/router';
 import { catchError, Observable, of } from 'rxjs';
 import { map } from 'rxjs/operators';
 
+// sonar-disable-next-statement (sonar doesn't read tsconfig paths entry)
+import { KeyedCache } from 'shared';
```

Comment:
There are 3 projects in the monorepo ('shared', 'metis' and 'sandbox').  Here I move the cache utility to the shared project so that the sandbox (as well as metis) can use it.

Reply:
done. I understood after our meeting.